docs/ftue/00_tour/FTUE_00_tour.md

```diff
@@ -0,0 +1,100 @@
+# Interface Basics
+
+Welcome to Griptape Nodes! This tutorial will guide you through your first steps with this powerful tool, from installation to creating your first workflow.
+
+## What You'll Learn
+
+In this tutorial, you will:
```

Comment:
ah, ran into the joys of some rendering complexity with mkdocs serve.. add a newline after the `:`, otherwise it's rendered like this:

![image](https://github.com/user-attachments/assets/2e0b01fc-4c08-4abe-b3c0-3f7e3c1da6ba)